rules/docker-dpu-base.dep

```diff
@@ -0,0 +1,4 @@
+DEP_FILES := rules/docker-dpu-base.dep rules/docker-dpu-base.mk
```

Comment:
Can you move docker-dpu-base.dep and docker-dpu-base.mk to platform/pensando folder?

Reply:
Done